native_mate/native_mate/function_template.h

```diff
@@ -156,7 +156,7 @@ struct ArgumentHolder {
   ArgumentHolder(Arguments* args, int create_flags) : ok(false) {
     if (index == 0 && (create_flags & HolderIsFirstArgument) &&
         Destroyable::IsDestroyed(args)) {
-      args->ThrowError("Object has been destroyed");
+      args->ThrowError("Object has been destroyed.");
```

Comment:
This error message has always annoyed me, can we say something somewhat user friendly here 😆 

```suggestion
      args->ThrowError("Attempted to perform an action on an Electron Object (like a BrowserWindow, Menu or other Electron object) but it has been destroyed.  You can check isDestroyed() before using it or ensure that you remove references to destroyed objects");
```

That's really long, and probably not what we want, but we can definitely do better there.

cc @felixrieseberg do you have any ideas for a handy error message here that's more useful?